robottelo/hosts.py

```diff
@@ -686,16 +686,20 @@ def virt_who_hypervisor_config(
             self.put_ssh_key(settings.server.ssh_key, hypervisor_ssh_key_name)
             # setup the ssh config and known_hosts files
             self.update_known_hosts(
-                self, hypervisor_ssh_key_name, hypervisor_hostname, user=hypervisor_user
+                hypervisor_ssh_key_name, hypervisor_hostname, user=hypervisor_user
             )
 
         # upload the virt-who config deployment script
         virt_who_deploy_directory = '/root/virt_who_deploy_output'
         virt_who_deploy_filename = f'{gen_alpha(length=5)}-virt-who-deploy-{config_id}'
         virt_who_deploy_file = f'{virt_who_deploy_directory}/{virt_who_deploy_filename}'
+        # create the virt-who directory on the broker VM
+        self.run(f'mkdir -p {virt_who_deploy_directory}')
+        # create the virt-who directory on satellite
+        satellite = Satellite(settings.server.hostname)
```

Comment:
If you're just using the current target sat, you can empty init the class

Reply:
@JacobCallahan Do you mean:

    -        satellite = Satellite(settings.server.hostname)

    +        satellite = Satellite()

Reply:
correct!

Reply:
OK, done that, now testing locally with new Satellite